src/payment/PaymentPage.jsx

```diff
@@ -105,7 +105,9 @@ class PaymentPage extends React.Component {
           />
         </div>
         <div className="col-md-7 pl-md-5">
-          <Checkout />
+          <Checkout
+            products={this.props.products}
```

Comment:
I believe you can get `products` from props in the Checkout component, so no need to pass this in here

Reply:
oh interesting, good call. i'll drop this then